sdk/storage/Azure.Storage.Webjobs.Extensions.Queues/src/Config/QueuesOptions.cs

```diff
@@ -36,7 +36,18 @@ public class QueuesOptions : IOptionsFormatter
         public QueuesOptions()
         {
             _newBatchThreshold = -1;
-            _processorCount = SharedUtility.GetProcessorCount();
+            _processorCount = GetProcessorCount();
+        }
+
+        private static int GetProcessorCount()
+        {
+            int processorCount = 1;
+            var skuValue = Environment.GetEnvironmentVariable(Constants.AzureWebsiteSku);
+            if (!string.Equals(skuValue, Constants.DynamicSku, StringComparison.OrdinalIgnoreCase))
```

Comment:
SkuUtility.IsDynamicSku?

Reply:
Should we move all DynamicSku related to one place?

Reply:
Let' me try and see what can be done.